hack/e2e/ensure_cmctl.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
A bit unsure about this tool, but if we happen to keep it, dont forget the license